helm/dagster/templates/helpers/_helpers.tpl

```diff
@@ -166,7 +166,7 @@ pyamqp://{{ .Values.rabbitmq.rabbitmq.username }}:{{ .Values.rabbitmq.rabbitmq.p
 rpc://
 {{- else if .Values.redis.enabled -}}
 {{- $password := ternary (printf ":%s@" .Values.redis.password) "" .Values.redis.usePassword -}}
-{{- $connectionUrl := printf "redis://%s%s:%g/%g" $password .Values.redis.host .Values.redis.port (.Values.redis.backendDbNumber | default 0) -}}
+{{- $connectionUrl := printf "redis://%s%s:%g/%g" $password .Values.redis.host .Values.redis.port (.Values.redis.backendDbNumber | default (float64 0)) -}}
```

Comment:
what did this hit

Reply:
it made a malformed URL with the default values :( (something like (!%g-int) in the middle of the URL)

added the test_default_redis_config test case above to cover it